plugins/queue/rabbitmq_amqplib.js

```diff
@@ -26,6 +26,7 @@ exports.rabbitmq_queue = function (next, connection) {
 
 exports.init_amqp_connection = function () {
     const cfg = this.config.get("rabbitmq.ini").rabbitmq;
+    const queueArgs = this.config.get("rabbitmq.ini").queue_args;
```

Comment:
This could be shortened to:

```suggestion
    const queueArgs = cfg.queue_args;
```

or omit it entirely and reference it from cfg below.